ibridges/__main__.py

```diff
@@ -323,10 +335,112 @@ def ibridges_list():
         default=None,
         nargs="?",
     )
+    parser.add_argument(
+        "-m", "--metadata",
+        help="Show metadata for each iRODS location.",
+        action="store_true",
+    )
 
     args, _ = parser.parse_known_args()
     with _cli_auth(ienv_path=_get_ienv_path()) as session:
-        _list_coll(session, _parse_remote(args.remote_path, session))
+        _list_coll(session, _parse_remote(args.remote_path, session), args.metadata)
+
+
+def ibridges_meta_show():
```

Comment:
To be consistent with the other iRODS clients we should  rename `show` into `list`.

Reply:
Done.